app/src/main/sentences/sl/open.dslf

```diff
@@ -0,0 +1,2 @@
+open: medium
+odpri|zaženi|pokaži|? (aplikacijo? .what.)|(.what. aplikacijo?);
```

Comment:
```suggestion
odpri|zaženi|pokaži (aplikacijo? .what.)|(.what. aplikacijo?);
```
I think this is what you want. `|?` is a syntax error, but leaving the `?` would have meant that any sentence would match, since any word except for the `.what.` capturing group would be optional. If you meant something else, maybe it would be a good idea to split this into multiple sentences instead.